SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -22,6 +22,7 @@ namespace ESettingsWorkerLogVerbosity
 {
 enum Type
 {
+	NoLogging,
```

Comment:
Just wondering why we previously started from value 1, perhaps we should start `NoLogging` from 1 instead?

Reply:
In the engine header file (LogVerbosity) the 'NoLogging' has value 0, so we can make it a 0 then?